### PR TITLE
Alt data sources

### PR DESCRIPTION
**Pull request by sloanelybutsurely**
Hey y'all. I wrote this post on alt's data sources.

WDYT?

@trunkclub/core @trunkclub/ui 

Here is the gist used for code examples https://gist.github.com/zperrault/dc158e5126cbabad2bf0

### CODE REVIEW

**Comment by jeffmeyers** [comment]
LGTM

**Comment by SuzIllini2** [comment]
Yeah yeah yeah!

File: _posts/2015-08-13-using-alts-data-sources.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+comments: true
+categories:
+  - alt
+  - react
+  - flux
+  - babel
+author: Zach Perrault
+---
+
+The Trunk Club tech team recently committed to using [Alt](http://alt.js.org) [for a year](https://twitter.com/peterpme/status/626190340844687361). One of the more interesting features of Alt are [data sources](http://alt.js.org/docs/async).
```
Comment (pdadhaniya) [inline _posts/2015-08-13-using-alts-data-sources.md]: I _think_ this should be `One of the more interesting features of Alt is` instead of `are`

Comment (sloanelybutsurely) [inline _posts/2015-08-13-using-alts-data-sources.md]: :100:

**Comment by MTTHWBSH** [comment]
unamused_kanye

**Pull request merged**